src/main/java/Revers.java

```diff
@@ -0,0 +1,31 @@
+import java.util.*;
+
+public class Revers {
```

Comment:
Please rename this and test class - it is not about reverse :)

Reply:
Renamed to Sorted

Reply:
methods in this class don't sort values either :)
I suggest to rename it to smth like CombineListsIntoMap or MapGenerator

Reply:
Renamed to MapGenerator